src/6_branch.js

```diff
@@ -922,7 +922,7 @@ Branch.prototype['link'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
  * ##### Example
  * ```js
  * branch.sendSMS(
- *     phone: '9999999999',
+ *     '9999999999',
```

Comment:
good! please run `make` after changing this, which will update docs/web/3_branch_web.md